.github/workflows/agent-build-new.yml

```diff
@@ -187,11 +193,9 @@ jobs:
         env:
           AWS_ACCESS_KEY: ${{ secrets.CT_AWS_DEV_EC2_ACCESS_KEY }}
           AWS_SECRET_KEY: ${{ secrets.CT_AWS_DEV_EC2_SECRET_KEY }}
-          AWS_PRIVATE_KEY_NAME: ${{ needs.pre_job.outputs.aws_private_key_name }}
-          AWS_PRIVATE_KEY_PATH: "/tmp/private_key.pem"
+          AWS_PRIVATE_KEY_NAME: ${{ secrets.CT_AWS_DEV_EC2_PRIVATE_KEY_NAME }}
+          AWS_PRIVATE_KEY: ${{ secrets.CT_AWS_DEV_EC2_PRIVATE_KEY }}
           AWS_REGION: ${{ needs.pre_job.outputs.aws_region }}
           CICD_WORKFLOW: ${{ needs.pre_job.outputs.cicd_workflow }}
         run: |
-          echo "${{ secrets.CT_AWS_DEV_EC2_PRIVATE_KEY }}" > "${AWS_PRIVATE_KEY_PATH}"
```

Comment:
I assume this is not needed anymore since the Python script only needs access to the AWS credentials available in the environment variables and not also access to private key, right?